concent_api/verifier/utils.py

```diff
@@ -334,7 +336,7 @@ def unpack_archives(file_paths, subtask_id):
             )
 
 
-def validate_downloaded_archives(subtask_id, archives_list):
+def validate_downloaded_archives(subtask_id: str, archives_list: Iterable[str]) -> None:
```

Comment:
Same here, how about changing `archives_list` to `*args` ?

Reply:
my pov is like above ;)